lib/contents.js

```diff
@@ -35,11 +31,11 @@ function contents(map, tight, prefix, ordered) {
 }
 
 // Insert an entry into `parent`.
+// eslint-disable-next-line max-params
 function insert(entry, parent, tight, prefix, ordered) {
-  var siblings = parent.children
-  var tail = siblings[siblings.length - 1]
-  var index = -1
-  var item
+  const siblings = parent.children
+  const tail = siblings[siblings.length - 1]
+  let index = -1
```

Comment:
Not ESM related.. but the distinction of which `var` became a `let` or `const` is interesting. Seeing the distinction like this is useful 👍🏻

Also, the scope of `item` changing from function scope to block scope 👀